tasks/update-granules-metadata/schemas/config.json

```diff
@@ -0,0 +1,72 @@
+{
+  "title": "UpdateGranulesMetadataConfig",
+  "description": "Describes the config used by the update-granules-metadata task",
+  "type": "object",
+  "required": [
+    "bucket",
```

Comment:
I also don't think config.bucket is used in the task code?

Reply:
It's not but I was running into a strange error when I removed it.